views/js/qtiCreator/widgets/helpers/validators.js

```diff
@@ -69,6 +69,22 @@ define([
                     throw 'missing required option "serial"';
                 }
             }
+        },
+        //Define a validator that check validity of a URL
+        {
+            name: 'isValidUrl',
+            message: __('Invalid URL'),
+            validate: function validate(value, callback) {
+                if(value) {
+                    try{
```

Comment:
I think the `try catch` is not necessary, because you can throw the error as well. Like above.